### ISSUE TITLE
Split by intent not working on cc.tides.coloredcow.com

### ISSUE THREAD

**Issue by Ranjana-G** [glific/glific#2278]
**Describe the bug**
There are 2 KB in the glificsmalltalk agent  each with own intent - see here 
https://dialogflow.cloud.google.com/#/agent/tides-saas-309509/knowledgeBases
Glific FAQ 
CSRFAQS

I designed a flow to detect both these intents and respond appropriately using split by intent node and printing the response field from the result. However Even if question is valid the flow alwasy goes to other. 
**To Reproduce**
1. Say Gyaani in simulator ( triggers Ask Any Question Flow) 
2. When bot says Ask me anything paste and FAQS questions
3. Note that response always comes from node attached to "other" 

**Screenshots**
https://drive.google.com/file/d/1uCN58PTU_iR7CzFVr3U8U80axiYNBHeA/view?usp=sharing
https://drive.google.com/file/d/1cB07bIAtHAGVOeLgDGys-8UmRA13UboZ/view?usp=sharing

used chrome on windows 8.1 and Ubuntu 18.04

**Comment by pankaj-ag** [comment]
So I did some research and found that. 
The knowledgeBase is a beta feature for Dialogflow and is only available for API version: **v2beta1**
Currently, on production, we have set up the stable API version which is: **v2** 

I checked and there are no breaking changes as such for the beta version. 

**Proposed Solution:** 
1. A new input in the Dialogflow provider config for the API version. (By default it will be default version v2)
2. That version input can be configurable by the Glific team only and we can use the **v2beta1** for CC and Danamojo.
3. I tested it and it seems to be working fine. 

@dlobo let me know if you have any suggestions on this. 


Reference
https://cloud.google.com/dialogflow/es/docs/reference/rpc/google.cloud.dialogflow.v2
